azure-servicebus/azure/servicebus/aio/async_message.py

```diff
@@ -0,0 +1,190 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#--------------------------------------------------------------------------
+
+import datetime
+import functools
+
+from azure.servicebus.common import message
+from azure.servicebus.common.utils import get_running_loop
```

Comment:
Note for myself and FYI to you: I shall create a azure.common.async_tools with it instead of keeping this SB only